Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM registry.access.redhat.com/openshift3/nodejs-010-rhel7
 
+RUN mkdir -p /opt/app-root/nodejs \
```

Comment:
Instead of installing node 6.3.1 we could use @ryanj images instead: https://github.com/ryanj/origin-s2i-nodejs in the FROM instruction.